commcare/src/main/java/org/motechproject/commcare/tasks/builder/model/FormWithApplicationName.java

```diff
@@ -0,0 +1,45 @@
+package org.motechproject.commcare.tasks.builder.model;
+
+import org.motechproject.commcare.domain.FormSchemaJson;
+
+/**
+ * Stores form and it's application name.
+ */
+
+public class FormWithApplicationName {
+    private FormSchemaJson form;
+    private String applicationName;
+
+    public FormWithApplicationName(FormSchemaJson form, String applicationName) {
+        this.form = form;
+        this.applicationName = applicationName;
+    }
+
+    public FormSchemaJson getForm() {
+        return form;
+    }
+
+    public void setForm(FormSchemaJson form) {
+        this.form = form;
+    }
+
+    public String getApplicationName() {
+        return applicationName;
+    }
+
+    public void setApplicationName(String applicationName) {
+        this.applicationName = applicationName;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Why aren't all fields included in the equals and hashcode methods? On a side note, you can use java7's Objects.equals to reduce complexity and retain the same effect